src/client/common/application/applicationEnvironment.ts

```diff
@@ -42,7 +42,7 @@ export class ApplicationEnvironment implements IApplicationEnvironment {
         }
     }
     public get userCustomKeybindingsFile(): string | undefined {
-        return path.resolve(this.extensionContext.globalStoragePath, '..', '..', 'keybindings.json');
+        return path.resolve(this.extensionContext.globalStorageUri.fsPath, '..', '..', 'keybindings.json');
```

Comment:
This old API was deprecated by VS Code a while ago, figured I might as well change this here.